scripts/exhafs_grid.sh

```diff
@@ -172,10 +174,29 @@ elif [ $gtype = regional ]; then
   # number of parent points
   nptsx=`expr $iend_nest - $istart_nest + 1`
   nptsy=`expr $jend_nest - $jstart_nest + 1`
+ # 
+  i_center=`expr $nptsx / 2`
+  j_center=`expr $nptsy / 2`
   # number of compute grid points
   npts_cgx=`expr $nptsx  \* $refine_ratio / 2`
   npts_cgy=`expr $nptsy  \* $refine_ratio / 2`
  
+  # number of new compute grodpoints
```

Comment:
@BantwaleEnyew-NOAA Could you please revert these changes, which should not be part of this PR?